src/AdoNet/Shared/Storage/DbConnectionFactory.cs

```diff
@@ -5,7 +5,19 @@
 using System.Linq;
 using System.Reflection;
 
-namespace Orleans.SqlUtils
```

Comment:
Could there be one main package that have these common parts to every other functionality which the persistence, reminders, statistics and membership implementations then reference? This way, it seems by reading this part, one could avoid some of the conditional namespaces.

Reply:
That was one another idea we had, having a "Core" package, but decided to share files instead.